optaweb-vehicle-routing-backend/src/main/java/org/optaweb/vehiclerouting/plugin/routing/RoutingConfig.java

```diff
@@ -64,7 +65,7 @@ GraphHopperOSM graphHopper() {
         }
         graphHopper.setOSMFile(osmPath);
         graphHopper.setGraphHopperLocation(GH_DIR + osmPath.replaceFirst(".*/(.*)\\.osm\\.pbf$", "$1"));
-        graphHopper.setEncodingManager(new EncodingManager("car"));
+        graphHopper.setEncodingManager(EncodingManager.create(FlagEncoderFactory.CAR));
```

Comment:
Is seems that the encoding is not backward compatible with an .osm file I have cached locally before this change:

springframework.beans.BeanInstantiationException: Failed to instantiate [com.graphhopper.reader.osm.GraphHopperOSM]: Factory method 'graphHopper' threw exception; nested exception is java.lang.IllegalStateException: Encoding does not match:
Graphhopper config: car|speed_factor=5.0|speed_bits=5|turn_costs=false|version=2
Graph: car|speed_factor=5.0|speed_bits=5|turn_costs=false|version=1
Change configuration to match the graph or delete local/graphhopper/belgium-latest/

But as the error message suggests the solution - deleting the old data - it's not an issue.

Reply:
Yes. Minor rectification: the incompatibility is not related to OSM data but the GraphHopper "graph" data that is a result of parsing OSM and is stored on file system as:
```
$ ls -1 local/graphhopper/belgium-latest
edges
geometry
location_index
names
nodes
nodes_ch_fastest_car
properties
shortcuts_fastest_car
```
So, this graph cache/storage version has changed between graphhopper 0.11 and 0.13. The solution to delete the graph data and let graphhopper regenerate it from OSM is acceptable.